stem/IO/kratos_material_io.py

```diff
@@ -380,8 +379,7 @@ def create_material_dict(
         # initialize material dictionary
         material_dict: Dict[str, Any] = {"model_part_name": f"{self.domain}.{part_name}",
                                          "properties_id": material_id,
-                                         "Material": {"constitutive_law": {"name": ""},
```

Comment:
is this no longer needed?

Reply:
not as default, spring damper and mass elements dont have a separate constitutive law in Kratos. (the law is incorporated in the element)